packages/cli/CommandLineHelp.md

```
@@ -360,4 +363,26 @@ Start the indexer without any automatic codegen
 
 
 
+## `envio config`
+
+Inspect the resolved indexer configuration
+
+**Usage:** `envio config <COMMAND>`
+
+###### **Subcommands:**
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h6

(MD001, heading-increment)